helm_deploy/create-and-vary-a-licence/values.yaml

```diff
@@ -33,6 +33,7 @@ generic-service:
     LICENCE_WATERMARK: "false"
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY);IngestionEndpoint=https://northeurope-0.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/"
     SERVICE_NAME: "create-and-vary-a-licence"
+    SHOW_WHATS_NEW_BANNER: "false"
```

Comment:
make SHOW_WHATS_NEW_BANNER flag true once Version changes are deployed.

Reply:
to test this you could override this on test1?

Then once this is merged we could see that dev doesn't have the banner but that test1 does.

Reply:
to override it in test1 - you can specify it in `values-test1.yml`